source/support/fob.sqf

```diff
@@ -105,15 +123,7 @@ publicVariable "Array_of_FOBS";
 Array_of_FOBname = Array_of_FOBname + [_fobname];
 publicVariable "Array_of_FOBname";
 
-
-
-sleep 1;
-
 saveGame;
 
-sleep 1;
-
 sleep 600;
-_art = [player1,"fob_support"] call BIS_fnc_addCommMenuItem;
-
-// Land_Cargo_HQ_V1_F
+_art = [player,"fob_support"] call BIS_fnc_addCommMenuItem;
```

Comment:
NEVER end a script with a variable assignment. That will trigger the only real SQF language bug in existence.